docs/developer/architecture/core/patterns-scoped-services.asciidoc

```diff
@@ -18,13 +16,11 @@ async function handler(context, req, res) {
 }
 ----
 
-The
-{kib-repo}blob/{branch}/docs/development/core/server/kibana-plugin-core-server.requesthandlercontext.md[request
-handler context] exposes the following scoped *core* services:
+The {kib-repo}/blob/8.9/packages/core/http/core-http-server/src/router/request_handler.ts[request handler context] exposes the following scoped *core* services:
```

Comment:
Replace main branch with concrete branch here and in other files this PR touches